lib/tokens.ts

```diff
@@ -171,94 +139,83 @@ export type ThemeV3WithDarkModeSupportType = {
 }
 export type AllThemesType = ThemeV3Type & ThemeV3WithDarkModeSupportType
 
+// These are for styled-system
+export type SpacingUnitsTheme = { space: Record<SpacingUnit, any> }
+export type ColorsTheme = { colors: Record<Color, any> }
+
+const THEME = {
+  ...mobileUsefulTHEME_V3,
+  colors: {
+    ...THEME_V3.colors,
+    background: THEME_V3.colors.white100,
+    onBackground: THEME_V3.colors.black100,
+    onBackgroundHigh: THEME_V3.colors.black100,
+    onBackgroundMedium: THEME_V3.colors.black60,
+    onBackgroundLow: THEME_V3.colors.black30,
+    surface: THEME_V3.colors.white100,
+    onSurface: THEME_V3.colors.black100,
+    onSurfaceHigh: THEME_V3.colors.black100,
+    onSurfaceMedium: THEME_V3.colors.black60,
+    onSurfaceLow: THEME_V3.colors.black5,
+    primary: THEME_V3.colors.black100,
+    onPrimaryHigh: THEME_V3.colors.white100,
+    onPrimaryMedium: THEME_V3.colors.black5,
+    onPrimaryLow: THEME_V3.colors.black5,
+    secondary: THEME_V3.colors.black30,
+    onSecondaryHigh: THEME_V3.colors.black100,
+    onSecondaryMedium: THEME_V3.colors.black60,
+    onSecondaryLow: THEME_V3.colors.black60,
+    brand: THEME_V3.colors.blue100,
+    onBrand: THEME_V3.colors.white100,
+    devpurple: "#6E1EFF",
+  },
+  space: convertWebSpacingUnitsToMobile(spaceNumbers),
+  textTreatments: convertWebTextTreatmentsToMobile(textVariantsWithUnits),
+  fonts: {
+    sans: {
+      regular: "Unica77LL-Regular",
+      italic: "Unica77LL-Italic",
+      medium: "Unica77LL-Medium",
+      mediumItalic: "Unica77LL-MediumItalic",
+    },
+  },
+}
+
+const THEME_LIGHT = THEME
```

Comment:
```suggestion
const v3light = THEME
```
this definitely needs the treatment below. we need to add the extra layer on v3light.